fiftyone/utils/transformers.py

```
@@ -1073,6 +1078,340 @@
+            self._output_processor._boxes = boxes
+            result = self._output_processor(outputs, image_sizes, self.config.confidence_thresh)
+            
+        except (RuntimeError, ValueError):
+            # Fallback to individual processing when batch processing fails
+            results = []
+            imgs_list = images if isinstance(images, list) else [images]
+            
+            for i, img in enumerate(imgs_list):
+                # Ensure we have a box for this image
+                if detection_boxes and i < len(detection_boxes):
+                    img_boxes = [detection_boxes[i]]
```

Pylint (3.3.7)

[error] Value 'detection_boxes' is unsubscriptable

(E1136)